docs/guides/README.md

```diff
@@ -13,7 +13,9 @@ parent:
   * [Keyring](./keys-wallets/keyring)
   * [Metamask](./keys-wallets/metamask)
```

Comment:
should we remove this one?

Reply:
Kept this one and merged the other into it